ui/admin/app/templates/scopes/scope/edit.hbs

```diff
@@ -66,21 +66,110 @@
   {{/if}}
 
   <page.body>
-    {{#if (can 'read model' @model)}}
-      <Form::Scope
-        @model={{@model}}
-        @submit={{route-action 'save' @model}}
-        @cancel={{route-action 'cancel' @model}}
-      />
-    {{else}}
-      <Rose::Layout::Centered>
-        <Rose::Message as |message|>
-          <message.description>
-            {{t 'resources.scope.messages.cannot_read'}}
-          </message.description>
-        </Rose::Message>
-      </Rose::Layout::Centered>
-    {{/if}}
+    <Rose::Layout::BodyContent as |bc|>
+      <bc.Body>
+        {{#if (can 'read model' @model)}}
+          <Form::Scope
+            @model={{@model}}
+            @submit={{route-action 'save' @model}}
+            @cancel={{route-action 'cancel' @model}}
+          />
+        {{else}}
+          <Rose::Layout::Centered>
+            <Rose::Message as |message|>
+              <message.description>
+                {{t 'resources.scope.messages.cannot_read'}}
+              </message.description>
+            </Rose::Message>
+          </Rose::Layout::Centered>
+        {{/if}}
+      </bc.Body>
+      {{#if (can 'attachStoragePolicy scope' @model)}}
+        <bc.Sidebar>
+          <div class='policy-sidebar'>
+            <Hds::Text::Display @tag='h3'>
+              {{t 'resources.policy.title'}}
+            </Hds::Text::Display>
+
+            <Hds::Text::Body @tag='p' class='description'>
+              {{t 'resources.policy.titles.description'}}
+            </Hds::Text::Body>
+
+            {{#if @model.storage_policy_id}}
+              <LinkListPanel as |P|>
+                <P.Item
+                  @icon='hourglass'
+                  @color='boundary'
+                  @text={{this.storage_policy.name}}
+                >
+                  {{#if this.storage_policy.retain_for.days}}
```

Comment:
This breaks if the policy is forever.

<img width="426" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/c62b1a56-d338-42d8-9614-428e3795d273">

Reply:
thanks, updated